### ISSUE TITLE
electrum can't connect to the network by default

### ISSUE THREAD

**Issue by unsigned-nerd** [subgraph/subgraph-os-issues#316]
Subgraph OS 1.0

When I ran Electrum for the first time and went through the GUI to perform the initial set up (seeds, etc), it couldn't connect to the network.  I checked the file `/var/lib/oz/cells.d/electrum.json` and found that the network setting was set to use `client, tcp, :9050`, just like other programs which could connect to the network.

But, if I configure the network setting inside Electrum GUI to use `SOCKS5, localhost, 9050`, Electrum can connect to the network.

**Comment by dma** [comment]
That's correct. We should include a default config as part of the skel for this.
